core/store/migrations/0043_add_uuid_to_pipeline_task_runs.go

```diff
@@ -0,0 +1,77 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up43 = `
+	-- Truncate tables to ease bigint -> UUID migration
+	TRUNCATE TABLE pipeline_runs, pipeline_task_runs, flux_monitor_round_stats_v2;
+
+	-- Migrate pipeline_task_runs to UUID
+	ALTER TABLE pipeline_task_runs DROP CONSTRAINT pipeline_task_runs_pkey; 
+	ALTER TABLE pipeline_task_runs DROP COLUMN id; 
+	ALTER TABLE pipeline_task_runs ADD COLUMN id uuid PRIMARY KEY;
+
+	-- Drop dependent constraints and indices
+	-- DROP INDEX flux_monitor_round_stats_v2_pipeline_run_id_idx;
+	-- DROP INDEX idx_unfinished_pipeline_task_runs;
+	-- DROP INDEX pipeline_task_runs_pipeline_run_id_dot_id_idx;
+	-- DROP CONSTRAINT flux_monitor_round_stats_v2_pipeline_run_id_fkey;
+	-- DROP CONSTRAINT pipeline_task_runs_pipeline_run_id_fkey;
+
+	-- ALTER TABLE pipeline_runs DROP CONSTRAINT pipeline_runs_pkey; 
+	-- ALTER TABLE pipeline_runs DROP COLUMN id; 
```

Comment:
Wouldn't dropping the id just autocascade to dropping all the dependent fks etc?

Reply:
I think it doesn't let you drop a column unless you drop the associated constraints/indices first.